packages/sdk-react/src/ui/Connect/connectors.ts

```diff
@@ -0,0 +1,47 @@
+import type { Connector, ConnectorList } from '../../types';
+
+export const FUEL_WALLET_CONNECTOR: Connector = {
+  name: 'Fuel Wallet',
+  image: '/connectors/fuel-wallet.svg',
+  connector: 'Fuel Wallet',
+  install: {
+    action: 'Install',
+    description: 'To connect your Fuel Wallet, install the browser extension.',
+    link: 'https://chrome.google.com/webstore/detail/fuel-wallet/dldjpboieedgcmpkchcjcbijingjcgok',
+  },
+  installed: false,
+};
+
+export const FUEL_WALLET_DEVELOPMENT_CONNECTOR: Connector = {
+  name: 'Fuel Wallet Development',
+  image: '/connectors/fuel-wallet-dev.svg',
+  connector: 'Fuel Wallet Development',
+  install: {
+    action: 'Install',
+    description:
+      'To connect your Fuel Wallet Development, install the browser extension.',
+    link: 'https://next-wallet.fuel.network',
```

Comment:
is it possible to send directly to extension link?

`https://chrome.google.com/webstore/detail/fuel-wallet-development/hcgmehahnlbhpilepakbdinkhhaackmc`